types/fingerprintjs__fingerprintjs/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
Where it is used? I cannot see this being required (it just adds dependency overhead)

---

I believe this is for `Promise`. As removing it introduces a type error. Is there a lighter alternative maybe?

---

Nope, the Promise IMO comes with `lib.es.*`:
https://github.com/microsoft/TypeScript/blob/master/lib/lib.es2015.promise.d.ts
I've run your PR tests with that change and it passes, there is no reference to DOM directly AFAIK